admin/app/services/session-info-service.js

```diff
@@ -52,27 +52,29 @@ export default Service.extend({
 
   buildSessionExportFileData(session) {
     return session.certifications.map((certification) => {
+      const certifIsRejected = certification.status === 'rejected';
```

Comment:
Qu'est ce que vous pensez de la [pratique](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig) qui consiste à préfixer les noms de variables contenant des booléens par `is` ?

Ici ça donnerait plutôt `isCertifRejected`.

Reply:
En effet je n'y avais pas pensé sur le coup, c'est mieux avec le is au début :)